platform/osx/SCsub

```diff
@@ -21,3 +21,17 @@ prog = env.add_program('#bin/godot', files)
 if (env["debug_symbols"] == "full" or env["debug_symbols"] == "yes") and env["separate_debug_symbols"]:
     env.AddPostAction(prog, run_in_subprocess(platform_osx_builders.make_debug_osx))
 
+# Create a ZIP archive so the export template can be used directly in Godot.
+zip_dir = env.Dir('#bin/.osx_zip')
+zip_files = env.InstallAs([
+    zip_dir.File('osx_template.app/Contents/Info.plist'),
+    zip_dir.File('osx_template.app/Contents/PkgInfo'),
+    zip_dir.File('osx_template.app/Contents/Resources/icon.icns'),
```

Comment:
Depending on the way MoltenVK driver will be shipped (there's `use_static_mvk` option), export template may contain additional `Resources/vulkan/icd.d/MoltenVK_icd.json` and `Frameworks/libMoltenVK.dylib` files.